extensions/rust/test/colorize-results/test_rs.json

```diff
@@ -154,19 +242,30 @@
 		}
 	},
 	{
-		"c": "println!",
-		"t": "source.rust support.function.std.rust",
+		"c": "println",
```

Comment:
In other languages (and in the old grammar for Rust), `print` statements are `support.function` should that be the case for Rust too?

Reply:
In Rust `println!` is a macro and not a function. I also think that `println!` maybe should be considered a single token and not two tokens (`println` and `!`)